src/backend/api/reviews.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+//Returns all reviews
```

Comment:
One tip would be to extract the handler and give that a name instead of the comment:
```js
router.get("/", listAllReviews);

async function listAllReviews(request, response) {
  const allReviews = await knex("review").select("*");
  response.json(allReviews);
}
```